content/tracing/advanced_usage/manual_tracing.md

```diff
@@ -33,6 +33,76 @@ public class MyClass {
 
 {{% /tab %}}
 {{% tab "Python" %}}
+
+If you would like to extend the functionality of the ``ddtrace`` library or gain
+finer control over instrumenting your application, several techniques are
+provided by the library.
+
+The following examples use the global tracer object which can be imported via:
+
+```python
+  from ddtrace import tracer
+```
+
+**Decorator**
+
+``ddtrace`` provides a decorator that can be used to trace a particular method
+in your application:
+
+```python
+  @tracer.wrap()
+  def business_logic():
+    """A method that would be of interest to trace"""
+    # ...
+    # ...
+```
+
+API details for the decorator can be found at [`ddtrace.Tracer.wrap()`][py_wrap]
+
+**Context Manager**
+
+To trace an arbitrary block of code, you can use the [`ddtrace.Span`][py_span]
+context manager:
+
+```python
+  import time
+
+  # trace some sleeping
+  with tracer.trace('time.sleep'):
+    # do interesting stuff besides sleeping for a second
+    time.sleep(1)
+```
+
+Further API details can be found at [`ddtrace.Tracer()`][py_tracer]
+
+**By Hand**
+
+If the above methods are still not enough to satisfy your tracing needs, a
+manual API is provided which will allow you to start and finish spans however
+you may require:
+
+```python
+  span = tracer.trace('interesting.stuff')
+
+  # do interesting stuff in between
+
+  # make sure to call span.finish() or the span will not be sent to Datadog
```

Comment:
`or the trace will not be sent`. We discard entirely the trace and not only this Span